relay-spans/src/span.rs

```diff
@@ -103,6 +107,23 @@ pub fn otel_to_sentry_span(otel_span: OtelSpan) -> EventSpan {
             };
             if key == "sentry.op" {
                 op = otel_value_to_string(value);
+            } else if key.starts_with("db") {
+                op = op.or(Some("db".to_string()));
```

Comment:
Maybe we should remove the support for `sentry.op` since I don't think it will be used given we'll start auto-detecting the op in Relay and with this, it means we could potentially overwrite the op after detecting it.

cc @AbhiPrasad

Reply:
yea this whole logic is a bit iffy because i stuck with one pass through the attributes array, it would be easier if this was a hashmap or we can also do multiple passes through the array to avoid the overwriting for cleaner logic.

but as a first version this is fine I think, we can iterate and clean up later.

Reply:
I don't think we can convert that as a hash map as it's the way the OTel schema is but multiple passes on the array would be cleaner indeed.